fpm/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+bundle install --path "${HOME}/bundles/${JOB_NAME}"
+
+cd "recipes/${RECIPE_NAME}/"
+
+bundle exec fpm-cook --target deb
+
+rsync -v pkg/*.deb deploy@apt-1.management:/tmp
+
```

Comment:
Should we add a function to clean up files and/or check if the file has already been packaged?